eslint.shared.config.mjs

```diff
@@ -25,35 +25,46 @@ export default [
     ]
   },
   {
-    name: '@o3r/report-unused-disable-directives',
+    name: '@o3r/framework/report-unused-disable-directives',
     linterOptions: {
       reportUnusedDisableDirectives: 'error'
     }
   },
   {
-    name: '@o3r/globals',
+    name: '@o3r/framework/settings',
+    settings: {
+      'import/resolver': 'node'
+    }
+  },
+  {
+    name: '@o3r/framework/ecma-version',
```

Comment:
Can be merged with `@o3r/framework/settings`, they are both configs to align with `tsconfig.base.json`